worlds/cvcotm/regions.py

```diff
@@ -0,0 +1,185 @@
+                                                lname.ug13,
+                                                lname.ug15,
+                                                lname.ug20],
+                                  "entrances": {"Underground Gallery Upper": "Gallery Lower to Upper"}},
+
+    "Underground Warehouse Start": {"locations": [lname.uw1],
+                                    "entrances": {"Underground Warehouse Main": "Into Warehouse Main"}},
+
+    "Underground Warehouse Main": {"locations": [lname.uw6,
+                                                 lname.uw8,
+                                                 lname.uw9,
+                                                 lname.uw10,
+                                                 lname.uw11,
+                                                 lname.uw14,
+                                                 lname.uw16,
+                                                 lname.uw16b,
+                                                 lname.uw19,
+                                                 lname.uw23,
+                                                 lname.uw24,
+                                                 lname.uw25]},
+
+    "Underground Waterway Start": {"locations": [lname.uy1],
+                                   "entrances": {"Underground Waterway Main": "Into Waterway Main"}},
+
+    "Underground Waterway Main": {"locations": [lname.uy3,
+                                                lname.uy3b,
+                                                lname.uy4,
+                                                lname.uy5,
+                                                lname.uy7,
+                                                lname.uy8,
+                                                lname.uy9,
+                                                lname.uy9b,
+                                                lname.uy12],
+                                  "entrances": {"Underground Waterway End": "Onward to Waterway End"}},
+
+    "Underground Waterway End": {"locations": [lname.uy12b,
+                                               lname.uy13,
+                                               lname.uy17,
+                                               lname.uy18]},
+
+    "Observation Tower": {"locations": [lname.ot1,
+                                        lname.ot2,
+                                        lname.ot3,
+                                        lname.ot5,
+                                        lname.ot8,
+                                        lname.ot9,
+                                        lname.ot12,
+                                        lname.ot13,
+                                        lname.ot16,
+                                        lname.ot20]},
+
+    "Ceremonial Room": {"locations": [lname.cr1,
+                                      lname.dracula]},
+}
+
+
+def get_region_info(region: str, info: str) -> Optional[List[str]]:
```

Comment:
This has two possible outputs
```suggestion
def get_region_info(region: str, info: str) -> Optional[Union[List[str], Dict[str, str]]]:
```